src/Features/Core/Portable/Completion/CompletionChange.cs

```diff
@@ -97,6 +111,16 @@ public sealed class CompletionChange
             return new CompletionChange(textChange, textChanges, newPosition, includesCommitCharacter);
         }
 
+        internal static CompletionChange Create(
+            TextChange textChange,
+            ImmutableArray<TextChange> textChanges,
+            string? lspSnippet,
+            int? newPosition = null,
+            bool includesCommitCharacter = false)
```

Comment:
internal constructors should not have optional values.  those optionals are on public APIs to make things nicer for external consumers, but we want to think about these values at all callsites so we don't accidentally get something by default that sin't desirable.